packages/flutter_tools/lib/src/commands/run.dart

```diff
@@ -94,6 +94,9 @@ class RunCommand extends RunCommandBase {
     argParser.addOption('use-application-binary',
         hide: !verboseHelp,
         help: 'Specify a pre-built application binary to use when running.');
+    argParser.addOption('kernel',
+        hide: !verboseHelp,
+        help: 'Path to a pre-built kernel blob to use when running.');
```

Comment:
this needs to say what a kernel blob is, and answer the question "why would i use this"